etc/taskcluster/decision_task.py

```diff
@@ -192,11 +217,9 @@ def linux_run_task(name, build_task, script):
     return (
         linux_task(name)
         .with_dockerfile(dockerfile_path("run"))
-        .with_early_script("""
-            ./etc/taskcluster/curl-artifact.sh ${BUILD_TASK_ID} target.tar.gz | tar -xz
-        """)
-        .with_env(BUILD_TASK_ID=build_task)
-        .with_dependencies(build_task)
+        .with_repo()
+        .with_curl_artifact_script(build_task, "target.tar.gz")
+        .with_script("tar -xzf target.tar.gz")
```

Comment:
Yes, all of this code is new and still a bit in flux. The “move curl’ing artifacts into decisionlib” commit could have been a separate PR, but since it’s merely nicer (IMO) code for doing the same thing I didn’t bother. (It is a separate commit though.)